### ISSUE TITLE
Plugin causing spread operator to fail in jsx

### ISSUE THREAD

**Issue by ksrb** [gajus/babel-plugin-react-css-modules#7]
**.babelrc**
```json
{
  "presets":[
    "react-app"
  ],
  "plugins":[
    [
      "react-css-modules",
      {
        "generateScopedName":"[path]___[name]__[local]___[hash:base64:5]"
      }
    ]
  ]
}
```

**index.js**
```javascript
var test = <div {...something}></div>;
```

**package.json**
Minimal package.json can be provided if necessary 'reference' [here](https://github.com/ksrb/babel-plugin-react-css-modules-test/blob/master/package.json).

**Error**
```
Module build failed: TypeError: xxx: Cannot read property 'name' of undefined
```

**Comment by PCreations** [comment]
@ksrb Will be fixed when PR #12 is merged

**Comment by ksrb** [comment]
Nice hopefully it'll get merged soon 😃 .
